.github/workflows/ci.yml

```
@@ -66,68 +132,294 @@ jobs:
+          XCRUN_FILE=$(ls -t "$DERIVED_DATA_PATH"/Build/Products/*.xctestrun | head -n 1)
+          python3 scripts/plan-tests.py audit-xctestrun \
+            --xctestrun "$XCRUN_FILE" --workspace-root "$GITHUB_WORKSPACE"
+
+      - name: Upload build products (consumed by every lane)
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-products
+          path: "${{ env.DERIVED_DATA_PATH }}"
+          retention-days: 1
+          if-no-files-found: error
+          exclude: |
```

actionlint (1.7.12)

[error] input "exclude" is not defined in action "actions/upload-artifact@v4". available inputs are "compression-level", "if-no-files-found", "include-hidden-files", "name", "overwrite", "path", "retention-days"

(action)